gap/Magic.gd

```diff
@@ -303,6 +303,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!         <Mark><A>gap_root_relative_path</A></Mark>
+#!         <Item>
+#!             Either true or a string representing a relative path. &GAPDoc;s MakeGAPDocDoc,
+#!             which is called to build the documentation, supports an optional parameter <A>gaproot</A>
+#!             that makes the links to other documentation entries relative. So this path, if present, should
```

Comment:
"other documentation entries" should perhaps be "other manuals" or "external documentation (such as the GAP reference manual)"